### PR TITLE
docs-concepts-overview-working-with-objects-labels-pr-2017-7-15

### PR DESCRIPTION
**Pull request by skytalemcc**
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to  
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

File: docs/concepts/overview/working-with-objects/labels.md [hunk 1]
```diff
@@ -6,47 +6,76 @@ redirect_from:
 - "/docs/user-guide/labels/"
 - "/docs/user-guide/labels.html"
 ---
```
Comment (markthink) [inline docs/concepts/overview/working-with-objects/labels.md:8]: Labels and Selectors 可译为 Labels 和 Selectors

File: docs/concepts/overview/working-with-objects/labels.md [hunk 2]
```diff
@@ -6,47 +6,76 @@ redirect_from:
...
-
+<!--
 _Labels_ are key/value pairs that are attached to objects, such as pods.
 Labels are intended to be used to specify identifying attributes of objects that are meaningful and relevant to users, but do not directly imply semantics to the core system.
 Labels can be used to organize and to select subsets of objects.  Labels can be attached to objects at creation time and subsequently added and modified at any time.
 Each object can have a set of key/value labels defined.  Each Key must be unique for a given object.
+-->
+_Labels_是联结到例如像pod这样的对象的键值对.
```
Comment (markthink) [inline docs/concepts/overview/working-with-objects/labels.md:15]: 联结 应为 连接

File: docs/concepts/overview/working-with-objects/labels.md [hunk 3]
```diff
@@ -6,47 +6,76 @@ redirect_from:
...
+标签是意在被用来指定那些对象(有意义的并和用户有关的)的确认属性,但并不是直接对核心系统隐含了语义.
```
Comment (markthink) [inline docs/concepts/overview/working-with-objects/labels.md:16]: but do not directly imply semantics to the core system. 译成：但并不是直接对核心系统隐含了语义
很不通顺，可以译：但对核心系统不要直接使用隐含语义.

另外建议对kubernetes 资源不要翻译..比如 Labels 可不译（此条不做强制要求）

**Comment by skytalemcc** [comment]
@markthink  已经提交了新的 请查看

**Pull request closed without merging**